index.html

```diff
@@ -1,467 +1,600 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" type="text/css" href="index.css" />
     <link rel="stylesheet" type="text/css" href="index3.css" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@200;400&display=swap"
-        rel="stylesheet" />
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400&display=swap" rel="stylesheet" />
     <link
-        href="https://fonts.googleapis.com/css2?family=MuseoModerno:wght@200&family=Roboto:wght@300;700&family=Smooch+Sans:wght@100&display=swap"
-        rel="stylesheet" />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@200;400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=MuseoModerno:wght@200&family=Roboto:wght@300;700&family=Smooch+Sans:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
     <link href="fonts/InaiMathi.ttf" rel="stylesheet" />
+    <script defer src="./index.js"></script>
     <title>Document</title>
-</head>
```

Comment:
- [ **OPTIONAL** ] For best practices and accessibility, I think  it would be better to update your project's title name to a meaningful name giving a short idea of your page like: `Assumpta's Portfolio`